client/containers/Pub/PubDocument/PubInlineSuggestedEdits.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useMemo } from 'react';
+import { Classes } from '@blueprintjs/core';
+
+import { buttons, FormattingBarSuggestedEdits } from 'components/FormattingBar';
+import { getSuggestionAttrsForNode } from 'components/Editor/plugins/suggestedEdits/operations';
+
+import { usePubContext } from '../pubHooks';
+
+require('./pubInlineMenu.scss');
+
+const shouldOpenBelowSelection = () => {
+	return ['Android', 'iPad', 'iPhone'].some((device) =>
+		navigator.userAgent.toLowerCase().includes(device.toLowerCase()),
+	);
+};
+
+const PubInlineSuggestedEdits = () => {
+	const { collabData, pubBodyState } = usePubContext();
+	const { editorChangeObject } = collabData;
+
+	const selection = collabData.editorChangeObject!.selection;
+
+	const shouldHide = useMemo(() => {
+		const selectionInSuggestionRange = (): boolean => {
+			if (!collabData.editorChangeObject || !collabData.editorChangeObject.view) return false;
+			const doc = collabData.editorChangeObject.view.state.doc;
+			let inSuggestionRange = false;
+			// inclusive range is sel-1, sel+1
+			doc.nodesBetween(selection.$anchor.pos - 1, selection.$head.pos + 1, (node) => {
+				if (inSuggestionRange) return;
+				const present = getSuggestionAttrsForNode(node);
+				if (present) inSuggestionRange = true;
+			});
+			return inSuggestionRange;
```

Comment:
I know I suggested this `nodesBetween` solution for looking through ranges initially (my bad!), but in this case I do think it's worth using the precomputed ranges in the suggested edits plugin state. And I think you should probably do that the same way the suggestion buttons determine if they should be enabled or not: https://github.com/pubpub/pubpub/blob/master/client/components/Editor/plugins/suggestedEdits/commands.ts#L35

If you call these functions without passing `dispatch`, the transaction never gets run, and they return false when the selection isn't inside a suggestion and true when it is.